docs/reference/data-streams/lifecycle/apis/get-global-retention.asciidoc

```diff
@@ -0,0 +1,89 @@
+[[data-streams-get-global-retention]]
+=== Get the global retention of data streams
+++++
+<titleabbrev>Get Data Stream Global Retention</titleabbrev>
+++++
+
+preview::[]
+
+Gets the global retention settings that apply on every data streams managed by data stream lifecycle.
+
+[[get-global-retention-api-prereqs]]
+==== {api-prereq-title}
+
+** If the {es} {security-features} are enabled, you must have the `monitor_data_stream_global_retention` or
+`manage_data_stream_global_retention` <<privileges-list-cluster,cluster privilege>> to use this API.
+
+[[data-streams-get-global-retention-request]]
+==== {api-request-title}
+
+`GET _data_stream/_global_retention`
+
+[[data-streams-get-global-retention-desc]]
+==== {api-description-title}
+
+Gets the global retention settings that applied on data streams managed by data stream lifecycle.
+
+[role="child_attributes"]
+[[get-global-retention-api-query-parms]]
+==== {api-query-parms-title}
+
+`local`::
```

Comment:
I think it would be good to document more about what this means and why I would ever want to set it to `true`. As a user I'd be thinking "Wait, this is *global* retention information, what does it even mean to ask for local global information?"

Reply:
I see, I added this parameter because it's something that the `MasterNodeReadRequest` provides. I do not expect a lot of users to use it, but I also do not have a strong argument to remove it or hide it. If we do it in other master requests why not here too.

I did look up the documentation of this parameter and I saw there is an existing resusable description so I changed it here. Let me know what you think.

Reply:
It sounds a little better. I'm a little uncomfortable that we're exposing it since I don't know why a user would want to do that, and we don't have any tests for it that I can find (including on the parent class). But no real objections.